### ISSUE TITLE
Comparator is not detecting `typedef` being removed

### ISSUE THREAD

**Issue by ahassany** [HuaweiDatacomm/yang-comparator#7]
I tried running the yang-comparator on [IETF Yang Types](https://github.com/YangModels/yang/blob/main/standard/ietf/RFC/ietf-yang-types%402013-07-15.yang) the version on the left was exact as defined in the RFC while the version on the right I edited and removed all the typedef statement from it. I was expecting the yang comparator to say these are non-backward-compatible versions. 

```
java -jar target/yang-comparator-1.1.0.jar -left --y v2  -right --y  v3 -o out/output.xml -compatible-check --rule src/main/resources/rules.xml
```

Output
```xml
<?xml version="1.0" encoding="UTF-8"?>

<modules/>
```

**Comment by lllyfeng** [comment]
yes. The changes of typedefs, groupings will be ignored in previous design.  I think these changes will really take effect in the nodes which reference them. Maybe we can change this algorithm, but it need time to estimate this change.
